api/user/user.service.js

```diff
@@ -1,36 +1,73 @@
-'use strict';
-
-import moment from "moment";
+import moment from 'moment';
 
 const users = [];
 
 class UserService {
     constructor() {
-        let nextId = 1;
-        while (nextId <= 10) {
+        this.nextId = 0;
+
+        // Seed with some dummy data.
+        for (let i = 1; i <= 10; i++) {
+            var id = this._generateNextId();
             users.push({
-                id: nextId,
-                name: 'User Name ' + nextId,
-                email: 'user-email-name' + nextId + '@test.com',
+                id: id,
+                name: `User Name ${id}`,
+                email: `user-email-name-${id}@test.com`,
                 createdAt: '2016-12-29'
```

Comment:
Not necessary but could be. Not focused on that right now. Taking into account for future changes.